src/app/core/services/api.service.ts

```diff
@@ -45,10 +45,13 @@ export class ApiService {
 		return this.http.get<Response<Cerf[]>>(HttpConfig.baseUrl + '/members/' + this.user._id + '/events');
 		// it is not the service's concern to clean up the data for the app to consume (e.g. reading 'success' or returning [])
 	}
-	getPendingCerfs() {
+	getPendingCerfs(clubId?: string) {
 		if(!this.user) return of(null);
+		if(this.user.access.club < 2) return of([]);	// general members have no business with submitted cerfs
```

Comment:
Prevent unauthorized access, which the back-end will throw an error if the member doesn't have correct access. Not really related to club/division